package.json

```diff
@@ -63,14 +60,17 @@
       "**/test/**/*.js"
     ],
     "testPathIgnorePatterns": [
-      "/node_modules/",
       "/test/fixtures/"
-    ]
+    ],
+    "moduleNameMapper": {
+      "^@nodesecure/sec-literal$": "@nodesecure/sec-literal/src/index.js",
+      "^estree-walker$": "estree-walker/src/index.js"
+    }
```

Comment:
Jest  an issue with module resolving when the `main` property of the package.json is missing. (Even if we know that we should use `exports` in esm)